vm/interpreter/src/gas_tracker/price_list.rs

```diff
@@ -322,6 +444,28 @@ impl PriceList {
     pub fn on_verify_seal(&self, _info: &SealVerifyInfo) -> GasCharge {
         GasCharge::new("OnVerifySeal", self.verify_seal_base, 0)
     }
+    #[inline]
+    pub fn on_verify_aggregate_seals(
+        &self,
+        aggregate: &AggregateSealVerifyProofAndInfos,
+    ) -> GasCharge {
+        let proof_type = aggregate.seal_proof;
+        let per_proof = self
+            .verify_aggregate_seal_per
+            .get(&proof_type)
+            .expect("There is an implementation error where proof type does not exist in table");
```

Comment:
If this is fallible, does it make sense to return a Result here?

Reply:
A result doesn't make sense since we expect there to be a per proof price when verifying an aggregate. If we wanted to get rid of the `.expect(...)` though it would be nice if the look up function for the price used the compiled to return a price for each enum variant.